### PR TITLE
Add Oceanic-Expanse compatibility

### PR DESCRIPTION
**Pull request by zeng-github01**
#324 

Need Code Review

### CODE REVIEW

File: src/main/java/thedarkcolour/futuremc/util/Integration.java
Comment (thedarkcolour) [inline src/main/java/thedarkcolour/futuremc/util/Integration.java:1]: This type of functionality does not belong in the `util` package. There is a separate `compat` package which has a system like the one you made already in place for a bunch of mods. It is also written in Kotlin.

Comment (thedarkcolour) [reply src/main/java/thedarkcolour/futuremc/util/Integration.java:1]: Additionally, I do not want to create an additional config file. This is also not the way I would prefer making the config, as using the annotations generates a user friendly screen while using this method does not.

Comment (thedarkcolour) [reply src/main/java/thedarkcolour/futuremc/util/Integration.java:1]: Add the option to FConfig

**Review by thedarkcolour — CHANGES_REQUESTED** [review]

File: src/main/java/thedarkcolour/futuremc/util/Integration.java
Comment (zeng-github01) [reply src/main/java/thedarkcolour/futuremc/util/Integration.java:1]: All OE installation judgments have been changed to use Compact.kt.
Testing in progress and uploading

About FConfig.kt Not sure what entries need to be added

Comment (zeng-github01) [reply src/main/java/thedarkcolour/futuremc/util/Integration.java:1]: My branch has submitted a new supplementary commit, but it doesn't seem to be displayed in the PR.

WHY？

Comment (zeng-github01) [reply src/main/java/thedarkcolour/futuremc/util/Integration.java:1]: Looks like this commit is finally synced

**Review by thedarkcolour — APPROVED** [review]

**Pull request merged**